lib/datasource/github.js

```diff
@@ -47,16 +44,14 @@ async function getDependency(purl) {
   if (!versions) {
     return null;
   }
-  // Filter by semver if no versionScheme provided
-  if (options.clean === 'true') {
-    versions = versions.map(version => version.replace(/^v(\d)/, '$1'));
-  }
   const dependency = {
     repositoryUrl: 'https://github.com/' + repo,
-    versions: {},
   };
-  versions.forEach(version => {
-    dependency.versions[version] = {};
-  });
+  // TODO: should Github receive the versionScheme as input?
+  // versions should be already sorted by GIT
```

Comment:
I don't believe that can be true. Git would need to be node semver-aware for that to be true. e.g. the Node.js github repo released 10.x a while ago yet 8.x still gets updates. I don't see how git would sort correctly using either (a) chronological order, or (b) alphabetical order.

I am OK with passing `versionScheme` to datasource and having it sort and/or filter based on that.